pkg/cmd/run.go

```diff
@@ -37,7 +37,6 @@ func addRunFlags(cmd *cobra.Command, cfg *config.MicroshiftConfig) {
 	cmd.MarkFlagFilename("config", "yaml", "yml")
 	// All other flags will be read after reading both config file and env vars.
 	flags.String("data-dir", cfg.DataDir, "The directory for storing runtime data.")
-	flags.String("audit-log-dir", cfg.AuditLogDir, "The directory for storing audit logs.")
```

Comment:
There's a sample configuration file in https://github.com/openshift/microshift/blob/main/packaging/microshift/config.yaml that has this config value, too. We can clean that up in a separate PR.

Reply:
Good catch. Opened https://github.com/openshift/microshift/pull/1056.